### PR TITLE
commands/group: add e2e tests batch 2

### PR DESCRIPTION
**Pull request by isacikgoz**
#### Summary
Add e2e tests for:
- `teamGroupEnableCmdF`
- `teamGroupDisableCmdF`
- `teamGroupStatusCmdF`
- `teamGroupListCmdF`

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-31414
https://mattermost.atlassian.net/browse/MM-31415
https://mattermost.atlassian.net/browse/MM-31416
https://mattermost.atlassian.net/browse/MM-31417

### CODE REVIEW

File: commands/group_e2e_test.go [hunk 1]
```diff
@@ -301,3 +301,209 @@ func (s *MmctlE2ETestSuite) TestChannelGroupListCmd() {
 		s.Require().Len(printer.GetErrorLines(), 0)
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestTeamGroupEnableCmd() {
```
Comment (mgdelacroix) [inline commands/group_e2e_test.go:305]: the tests here correspond to the DisableCmd function

File: commands/group_e2e_test.go [hunk 2]
```diff
@@ -301,3 +301,209 @@ func (s *MmctlE2ETestSuite) TestChannelGroupListCmd() {
...
+		s.Require().Len(printer.GetLines(), 0)
+		s.Require().Len(printer.GetErrorLines(), 0)
+
+		tm, appErr := s.th.App.GetTeam(team.Id)
+		s.Require().Nil(appErr)
+		s.Require().False(tm.IsGroupConstrained())
+	})
+}
+
+func (s *MmctlE2ETestSuite) TestTeamGroupDisableCmd() {
```
Comment (mgdelacroix) [inline commands/group_e2e_test.go:345]: and the other way around :P these tests are testing the EnableCmd command

**Review by mgdelacroix — APPROVED** [review]
LGTM! :+1:
Added a couple of nonblocking comments. //ccing @lindy65 for visibility on test ids

**Comment by lindy65** [comment]
Hi @isacikgoz - looks good to me :)  I've added the corresponding tests to Zephyr (TM4J).  Please help add corresponding Zephyr test code to the `sRun` lines of the E2E tests as per below.  Let me know once done and I'll go ahead and re-review/approve this PR :)  Thanks!

image

**Review by lindy65 — APPROVED** [review]
Thanks for adding the TM4J test codes @isacikgoz 👍 Looks good to me :)

**Review by mkraft — APPROVED** [review]

**Pull request merged**